website/docs/docs/integrate/dbt-semantic-layer.md

```diff
@@ -0,0 +1,132 @@
+
+&mdash; **What?** Public preview provides early access to new features, is supported and production ready, but not priced yet. Pricing for the dbt Semantic Layer will be introduced alongside the General Available (GA) release. There may also still be additions or modifications to product behavior. 
+
+&mdash; **When?** Public preview will end once the dbt Semantic Layer is available for GA. After GA, the dbt Semantic Layer will only be available to dbt Cloud **Team** and **Enterprise** plans.
+
+&mdash; **Where?** Public preview is enabled at the account level so you don’t need to worry about enabling it per user.
+
+&mdash; **Why?** Public preview is designed to test the functionality and collect feedback from the community on performance, usability, and documentation. 
+
+## Product architecture 
+
+The dbt Semantic Layer product architecture includes four primary components, and you can review that and its plan availability in more detail below:
+
+| Components | Information | Developer plans | Team plans | Enterprise plans | License |
+| --- | --- | :---: | :---: | :---: | --- |
+| **dbt metrics** | Allows you to define metrics in dbt Core. | ✅ | ✅ |  ✅  | Open source, Core |
+| **dbt Server**| HTTP server that is able to quickly compile metric queries per environment using dbt project code. | ✅ | ✅ | ✅ | BSL |
+| **SQL Proxy** | Reverse-proxy that accepts dbt-SQL (SQL + Jinja like query models and metrics, use macros), compiles the query into pure SQL, and executes the query against the data platform. | ✅ <br></br>_* Available during public preview only_ | ✅ | ✅ | Proprietary, Cloud (Team & Enterprise) |
+| **Metadata API**  | Accesses metric definitions primarily via integrations and is the source of truth for objects defined in dbt projects (like models, macros, sources, metrics). The Metadata API is updated at the end of every dbt Cloud run. | ❌ | ✅ | ✅ | Proprietary, Cloud (Team & Enterprise |
+
+dbt Semantic Layer integrations will:
+
+- Leverage the Metadata API to fetch a list of objects and their attributes, like metrics
+- Generate a dbt-SQL statement
+- Then query the SQL proxy to evaluate the results of this statement
+
+
+## Manage metrics
+
+:::info 📌
+
+New to dbt or metrics? Check out our [Getting Started guide](/guides/getting-started) to build your first dbt project! If you'd like to define your first metrics, try our [Jaffle Shop](https://github.com/dbt-labs/jaffle_shop_metrics) example project.
+
+:::
+
+If you're not sure whether to define a metric in dbt or not, ask yourself the following: 
+
+> *Is this something our teams consistently need to report on?*  
+
+An important business metric should be:
+
+- Well-defined (the definition is agreed upon throughout the entire organization)
+- Time-bound (able to be compared across time)
+
+A great example of this is **revenue** - it can be aggregated on multiple levels (weekly, monthly, etc) and is key for the broader business to understand:
```

Comment:
```suggestion
A great example of this is **revenue** &mdash it can be aggregated on multiple levels (weekly, monthly, etc) and is key for the broader business to understand:
```